src/pages/journal/__tests__/journal.effects.spec.ts

```diff
@@ -30,38 +36,136 @@ export class TestActions extends Actions {
+      expect(slotProvider.detectSlotChanges).toHaveBeenCalledWith({}, JournalProviderMock.mockJournal);
+      expect(slotProvider.extendWithEmptyDays).toHaveBeenCalled();
+      expect(slotProvider.getRelevantSlots).toHaveBeenCalled();
+      expect(result instanceof journalActions.LoadJournalSuccess).toBe(true);
+      done();
+    });
+
   });
+
+  it('should dispatch the failure action when the journal fails to load', (done) => {
+    // ARRANGE
+    spyOn(journalProvider, 'getJournal').and.throwError;
+    spyOn(slotProvider, 'detectSlotChanges').and.callThrough();
+    spyOn(slotProvider, 'extendWithEmptyDays').and.callThrough();
+    spyOn(slotProvider, 'getRelevantSlots').and.callThrough();
+    // ACT
+    actions$.next(new journalActions.LoadJournal());
+    // ASSERT
+    effects.loadJournal$.subscribe((result) => {
+      expect(journalProvider.getJournal).toHaveBeenCalled();
+      expect(slotProvider.detectSlotChanges).toHaveBeenCalledTimes(0);
+      expect(slotProvider.extendWithEmptyDays).toHaveBeenCalledTimes(0);
+      expect(slotProvider.getRelevantSlots).toHaveBeenCalledTimes(0);
+      expect(result instanceof journalActions.LoadJournalFailure).toBe(true);
+      done();
+    });
+
+  });
+
+  it('should dispatch the SetSelectedDate action with the correct date in the select next day effect', (done) => {
+    // ARRANGE
+    const selectedDate: string = DateTime.now().format('YYYY-MM-DD'); // Today
+    const nextDay: string = DateTime.at(selectedDate).add(1, Duration.DAY).format('YYYY-MM-DD'); // Tomorrow
+    store$.dispatch(new journalActions.LoadJournalSuccess(journalSlotsDataMock)); // Load in mock journal state
+    spyOn(store$, 'dispatch');
+    // ACT
+    actions$.next(new journalActions.SelectNextDay());
+    // ASSERT
+    effects.selectNextDayEffect$.subscribe((result) => {
+      expect(store$.dispatch).toHaveBeenCalledWith(new journalActions.JournalNavigateDay(nextDay));
+      expect(result).toEqual(new journalActions.SetSelectedDate(nextDay));
+      done();
+    });
+
+  });
+
+  it('should dispatch the SetSelectedDate action with the correct date in the select previous day effect', (done) => {
+    // ARRANGE
+    const selectedDate: string = DateTime.now().format('YYYY-MM-DD'); // Today
+    const nextDay: string = DateTime.at(selectedDate).add(1, Duration.DAY).format('YYYY-MM-DD'); // Tomorrow
+    store$.dispatch(new journalActions.LoadJournalSuccess(journalSlotsDataMock)); // Load in mock journal state
+    store$.dispatch(new journalActions.SetSelectedDate(nextDay));
+    spyOn(store$, 'dispatch');
+    // ACT
+    actions$.next(new journalActions.SelectPreviousDay());
+    // ASSERT
+    effects.selectPreviousDayEffect$.subscribe((result) => {
+      expect(store$.dispatch).toHaveBeenCalledWith(new journalActions.JournalNavigateDay(selectedDate));
+      expect(result).toEqual(new journalActions.SetSelectedDate(selectedDate));
+      done();
+    });
+
+  });
```

Comment:
This looks really neat. I like how each test is commented with arrange, act and assert. Very simple to follow.